mindsdb/integrations/handlers/timegpt_handler/timegpt_handler.py

```diff
@@ -88,6 +91,7 @@ def predict(self, df, args={}):
 
             # TODO: enable this post-refactor
             # X_df=None,  # exogenous variables
+            # add_history=False,  # insample
```

Comment:
nitpick: remove as it is added in line 48 already

Reply:
removed 'add_history=False' line